IMC_WebDev/FinalBlog.md

```diff
@@ -0,0 +1,224 @@
+
+Sometimes, objects have “children." For example, since our buildings are placed on the ground, the buildings are the “children” of the ground.
+You can see that “building1” is placed within the “ground” tag in our code.
+
+Our “building1” also has children of its own. Since walls and a roof are components of our building, we place them inside our &lt;building1&gt; tag. We repeat this whenever components belong to each other. (You can also see this with the grass area and trees.)
+
+After we create all of our objects and their “children,” it is time to close our body and our html tag by using /body and /html.
+
+When it’s time to design our objects in CSS, we MUST use the exact name we created for them in our HTML. For example, if I want to change the color of the bridge in my city, I need to name it “bridge” in my CSS, just like how it is in my HTML.
+
+Next, let’s prepare our editor for the CSS portion of the code. We do this by establishing a style tag. Since we will be using CSS, our style tag will look like this: 
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>Mini City Webinar</title>
+    	<style TYPE="text/css">
+```
+
+
+We will be writing all of our CSS in the head portion of our code.
+
+To style our webpage, we will need to learn the CSS syntax. A CSS rule-set consists of a selector and a declaration block.
+<img src=https://lh3.googleusercontent.com/3UcaVdjNU8MuMn4Feims4asq1epCgIOEwJX2goOH8SZE2eP2IKz5vRvCuSL9howvCc_Co_G2TE8Tqe5pIPgTAITV9KB28hqPEWXLzvZZ7xTnF1GNBohcY75WK00KBHIfbJ4nufVR>
+
+Apart from color and font-size, these are some of the properties you will need to get started:
+
+1. Position: This property specifies the type of positioning method used for an element, such as static, relative, fixed, absolute, or sticky. 
+* Static: Static positions are always positioned according to the normal flow of the page.
+* Relative: Relative positions are positioned relative to another object in your CSS.
+* Fixed: Elements with fixed positions will always stay in the same place no matter which way the page is scrolled.
+* Absolute: Similar to fixed, but absolute positions will always stay in the same place relative to another object, rather than the page.
+* Sticky: An element is positioned based on the user's scroll position.
+2. Display: This property affects whether or not an element is shown. If you choose display: none, then the element will not be visible.
+3. Background-color: It determines the background color of the element. A color can be specified by a valid color name (“black”), a hexadecimal value (like “ffffff”), or an rgb value (“rgb(255,0,0)”).
+4. Height and Width: These properties are used to set the height and width of the element.
+5. Transform: This property is used to make 2D transformations. The most commonly used transformations are rotate() and skew.
```

Comment:
Does rotate need to have () after it? If not, I would eliminate them. (Also, this goes for the two bullet points below, with rotate and skew too).